public_html/extensions/fast_checkout/storefront/view/default/template/pages/product/add_to_cart_buttons.tpl

```diff
@@ -0,0 +1,13 @@
+<?php // default addToCart button (for default checkout process) ?>
```

Comment:
There is no buynow in the default checkout. To avoid confusion, please remove comment in the baskets.

Reply:
done